utilities/responses.py

```diff
@@ -25,19 +25,21 @@ def send(self):
 
 
 class ErrorResponse(BaseResponse):
-    def __init__(self, dev_error=None, errors=None, **kwargs):
+    def __init__(self, message, dev_error=None, errors=None, **kwargs):
         super().__init__(**kwargs)
+        self.message = message
         self.dev_error = dev_error
         self.errors = errors
+        self.status = kwargs.get("status", 400)
 
 
 class SuccessResponse(BaseResponse):
-    def __init__(self, message, data=None, **kwargs):
+    def __init__(self, data=None, **kwargs):
         super().__init__(**kwargs)
-        self.message = message
```

Comment:
what happened to message?

Reply:
Since it could be None, it's passed through kwargs and initialized in `BaseResponse`. Albeit maybe it's not good idea since the `message` field is initialized in each child class differently and inheritance is meaningless. When I moved the `message` initialization to `BaseResponse`, I thought that it could be None in each child class.

Reply:
Sorry I didn't see that, I need to test it manually because we have no test case, I will reply to you asap.

Reply:
![image](https://user-images.githubusercontent.com/24853386/117586091-307adb00-b12b-11eb-88a6-5bea67a145f9.png)

Reply:
this is not our expected result :)

Reply:
please test your code before any other commit or pull request

Reply:
Actually I can't load test data to my DB. I'll try again.
Thanks